portfolio/src/main/webapp/script.js

```diff
@@ -31,9 +31,20 @@ function addRandomGreeting() {
  * fetchs /data information async aka Comments
  */
 async function getComments() {
-  const response = await fetch('/data');
+  //passes in user input on number of comments to display
+  var x = document.getElementById("maxcom");
```

Comment:
Should probably use a more descriptive variable name than |x| since it's not just |i| or |j| for an iterator.